tests/test_cli/test_install.py

```diff
@@ -94,10 +96,6 @@ def setup_class(cls):
                     "version": "==0.1.0",
                     "index": "https://test.pypi.org/simple",
                 },
-                "this_is_a_test_dependency_on_git": {
```

Comment:
Why removed?

Reply:
it used for test that should fail on the first dependency, but when i pass all dependencies to pip, pip asks for credentials for git, cause it tries to get access to this repo

Reply:
^ yes that's worth trying. Although do we always want it to be installed in the same environment? What about the multiprocess case?

Reply:
will check pip direct usage.

about venv - shouldn't be a problem, we can call any code inside venv, code is ready.


about venv support in MAM, it's not enabled yest as i remember, think it's better to ensure MAM multiprocess is completely ready - integrate with GUI